src/test/bin/winkernel/control.cpp

```diff
@@ -79,6 +79,10 @@ QuicTestCtlInitialize(
     WDF_IO_QUEUE_CONFIG QueueConfig;
     WDFQUEUE Queue;
 
+#ifdef QUIC_BUILD_STATIC
```

Comment:
This change actually isn't necessary, because we will never support kernel mode statically built. There are many more issues with doing that.